microsite/sidebars.json

```diff
@@ -212,6 +212,7 @@
       "plugins/structure-of-a-plugin",
       "plugins/integrating-plugin-into-software-catalog",
       "plugins/integrating-search-into-plugins",
+      "plugins/customization",
```

Comment:
Perhaps we could leave the documentation page hidden until this is more stable API? i.e. skip this change.